.github/workflows/ci.yml

```diff
@@ -0,0 +1,59 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
```

Comment:
Could you change this so that it runs on all branches? I'd like to have it in PRs as well